### PR TITLE
Adds API call for Ratings Snapshot, fixes styling for AirBnB, and enter production mode

### PR DESCRIPTION
**Pull request by jayrbarr**
This is a large pull request. I was debating keeping this all on a separate branch for deployment. But there are other changes that are beyond just deployment. So I'm merging it all to master.

### CODE REVIEW

**Comment by jayrbarr** [comment]
Apologies for the large pull request. @JasperC1999 @timothydadams  --- I hope I can do some more optimization in the morning before our presentation.

File: client/components/App.jsx [hunk 1]
```diff
@@ -142,7 +209,7 @@ const App = (props) => {
       <Grid>
         <ReviewsTitle>Reviews</ReviewsTitle>
         <div><ButtonRed buttonText="Write A Review" mouseClick={()=>{}} /></div>
-        <RatingSnapshot />
+        <RatingSnapshot fiveStars={fiveStars} fourStars={fourStars} threeStars={threeStars} twoStars={twoStars} oneStars={oneStars} />
```
Comment (JasperC1999) [inline client/components/App.jsx]: why were these props added?

File: client/components/AverageRatings.jsx [hunk 2]
```diff
@@ -10,10 +12,16 @@ const Title = styled.p`
...
-  </Container>
-)
+const AverageRatings = () => {
+  const bars = [{name: "Graphics", rating: 3.5}, {name: "Gameplay", rating: 4.2}, {name: "Lasting Appeal", rating: 4.7}];
+
+  return (
+    <Container>
+      <Title>Average Customer Ratings</Title>
+      <AverageStars rating="4.8" />
+      {bars.map((bar, index) => <AverageBars key = {index.toString()} name={bar.name} rating={bar.rating} />)}
```
Comment (JasperC1999) [inline client/components/AverageRatings.jsx]: why does the index need to be a string?

File: client/components/AverageRatings.jsx [hunk 3]
```diff
@@ -10,10 +12,16 @@ const Title = styled.p`
   font-weight: bold;
 `;
 
-const AverageRatings = () => (
-  <Container>
-    <Title>Average Customer Ratings</Title>
-  </Container>
-)
+const AverageRatings = () => {
+  const bars = [{name: "Graphics", rating: 3.5}, {name: "Gameplay", rating: 4.2}, {name: "Lasting Appeal", rating: 4.7}];
```
Comment (JasperC1999) [inline client/components/AverageRatings.jsx]: why are these items fixed values?

File: public/main.js [hunk 4]
```diff
@@ -1082,26 +1182,28 @@ var Title = styled_components__WEBPACK_IMPORTED_MODULE_1__["default"].p.withConf
...
     stars: 1,
-    total: 33
+    total: oneStars
   }];
-  var overallTotal = rows.reduce(function (sum, row) {
-    return sum + row.total;
-  }, 0);
   rows.forEach(function (row) {
     row.width = (row.total / overallTotal).toString();
   });
```
[the remark this answers is not in the document]
Comment (jayrbarr) [reply public/main.js]: yeah, why is that showing up again? sheesh

File: client/components/App.jsx (hunk 1 above)
Comment (jayrbarr) [reply client/components/App.jsx]: It was static before. Now they come in with API calls.

File: client/components/AverageRatings.jsx (hunk 2 above)
Comment (jayrbarr) [reply client/components/AverageRatings.jsx]: key seems to require that. Have you not run into that? I can't seem to get around that key error any other way.

File: client/components/AverageRatings.jsx (hunk 3 above)
Comment (jayrbarr) [reply client/components/AverageRatings.jsx]: Ah, you caught me. It's my last static component. I just ran out of time to do it all. But I wanted something in that spot. I'm also still missing filters, helpful votes, and commenting. It was a rather large ambitious module actually.

File: client/components/App.jsx (hunk 1 above)
Comment (JasperC1999) [reply client/components/App.jsx]: I see, for some reason I couldn't find the commit that had that.

File: client/components/AverageRatings.jsx (hunk 3 above)
Comment (JasperC1999) [reply client/components/AverageRatings.jsx]: They told us before not to try to copy everything from the website. You definitely had a lot to deal with on your component. I'm impressed with everything you did do!

File: client/components/AverageRatings.jsx (hunk 2 above)
Comment (JasperC1999) [reply client/components/AverageRatings.jsx]: I think I did but I can't remember it well

**Pull request merged**